draft-ietf-tls-esni.md

```diff
@@ -458,9 +458,9 @@ otherwise.
 
 To offer ECH, the client first chooses a suitable ECH configuration. To
 determine if a given `ECHConfig` is suitable, it checks that it supports the KEM
-algorithm identified by `ECHConfig.kem_id`, at least one KDF/AEAD algorithm
-identified by `ECHConfig.cipher_suites`, and the version of ECH indicated by
-`ECHConfig.version`. Once a suitable configuration is found, the client selects
+algorithm identified by `ECHConfig.contents.kem_id`, at least one KDF/AEAD algorithm
+identified by `ECHConfig.contents.cipher_suites`, and the version of ECH indicated by
+`ECHConfig.contents.version`. Once a suitable configuration is found, the client selects
```

Comment:
Do these lines break 80 characters?

Reply:
Yep, good catch! Reflowing the paragraph in a new commit.